jdi-light-mobile-tests/src/main/java/pseudo/site/webpage/ExtendedWebPage.java

```diff
@@ -0,0 +1,450 @@
+package pseudo.site.webpage;
+
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.DataList;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.DropdownExpand;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.*;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import org.openqa.selenium.WebElement;
+import pseudo.site.section.CustomSection;
+import pseudo.site.section.SomeData;
+
+import java.util.List;
+
+public class ExtendedWebPage extends ExtendWebPage {
+    @FindBy(id = "webElementPackage")
+    WebElement webElementPackage;
+    @FindBy(id = "webElementPublic")
+    public WebElement webElementPublic;
+    @FindBy(id = "webElementPrivate")
+    private WebElement webElementPrivate;
+    @UI(".webElementPackageUI")
+    WebElement webElementPackageUI;
+    @UI(".webElementPublicUI")
+    public WebElement webElementPublicUI;
+    @Css(".webElementPackageCss")
+    WebElement webElementPackageCss;
+    @XPath("//*[@class='webElementPublicXPath']")
+    public WebElement webElementPublicXPath;
+
+    @FindBy(id = "listWebElementPackage")
+    List<WebElement> listWebElementPackage;
+    @FindBy(id = "listWebElementPublic")
+    public List<WebElement> listWebElementPublic;
+    @FindBy(id = "listWebElementPrivate")
+    private List<WebElement> listWebElementPrivate;
+    @UI(".listWebElementPackageUI")
+    List<WebElement> listWebElementPackageUI;
+    @UI(".listWebElementPublicUI")
+    public List<WebElement> listWebElementPublicUI;
+    @Css(".listWebElementPackageCss")
+    List<WebElement> listWebElementPackageCss;
+    @XPath("//*[@class='listWebElementPublicXPath']")
+    public List<WebElement> listWebElementPublicXPath;
+
+    @FindBy(id = "uielementPackage")
+    UIElement uielementPackage;
+    @FindBy(id = "uielementPublic")
+    public UIElement uielementPublic;
+    @FindBy(id = "uielementPrivate")
+    private UIElement uielementPrivate;
+    @UI(".uielementPackageUI")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)